R/checks.R

```diff
@@ -789,4 +789,59 @@ check_fracwith_diswith <- function(form, rep) {
     filter(ptstatus == 1 & (num_of_fractures > 1 |
                               location_of_fracture != location_of_dislocation))
   return(problems)
+}
+
+#' Check if the number of fractures, if any, is valid for the set form5.x
+#' Also considers the logic of related selections for this check
+#' 
+#' @param form dataframe containing ptstatus and one set of form5.x
+#' @param rep which set of form5.x is checked
+#' @return a dataframe containing problematic entries
+#' 
+#' @import tidyverse
+#' @export 
+
+check_injwith_frac <- function(form, rep) {
```

Comment:
For consistency's sake, let's use abbreviations only when they are defined in the coding table.